pwa-devdocs/README.md

```diff
@@ -23,13 +23,13 @@ Use the following instructions to build a local instance of the documentation.
     ```
     cd pwa-studio
     ```
-3.  Run the following command to install the required node modules for the project:
+3.  Change into the **pwa-devdocs** directory using the following command:
     ```
-    npm install
+    cd packages/pwa-devdocs
     ```
-4.  Change into the **pwa-devdocs** directory using the following command:
+4.  Run the following command to install the required node modules for the project:
     ```
-    cd packages/pwa-devdocs
+    npm install
```

Comment:
Does it not work the way it was previously described?

We have a [Dependency Management](https://github.com/magento-research/pwa-studio/wiki/Dependency-Management) doc in the Wiki that states:

> The NPM scripts are designed so that nearly all operations, even ones for individual packages, should be run from the root directory.

Reply:
The doc site is a unique project in the mono repo and is self contained. So running `npm install` inside the docs directory makes sense and is the correct approach in my opinion.